src/playback/qml/MuseScore/Playback/internal/MeasureAndBeatFields.qml

```diff
@@ -66,6 +70,11 @@ Item {
 
                 font: root.font
 
+                navigation.panel: root.navigationPanel
+                navigation.order: root.navigationOrderStart
+                navigation.name: "measure"
+                accessible.name: qsTrc("global", "Measure", "Measure number")
```

Comment:
"engraving" maybe?

Reply:
On second thoughts, I think I'll go for `playback`:
- that makes it clearer where the string is to be found in the UI
- that makes it clearer that it is used here _not_ as just the noun "Measure" but as "Measure _number_". 

(and having the same string in multiple contexts doesn't create more work for translators anyway, so I'll go for `playback`.)